test/acceptance/tests.py

```diff
@@ -78,7 +78,7 @@ class OpenAssessmentTest(WebAppTest):
     }
 
     SUBMISSION = u"This is a test submission."
-    LATEX_SUBMISSION = u"[mathjaxinline]( \int_{0}^{1}xdx \)[/mathjaxinline]"
+    LATEX_SUBMISSION = u"[mathjaxinline]( \int_{0}^{1}xdx )[/mathjaxinline]"
```

Comment:
This change isn't strictly needed, but the red-highlighted text when debugging the test led me in the wrong direction initially, so I thought I'd fix it up. This final `)` does *not* need to be escaped:

<img width="176" alt="screen shot 2016-02-22 at 10 01 00 am" src="https://cloud.githubusercontent.com/assets/7373924/13222210/f347ccf4-d94c-11e5-9bbf-a2e7c763ef35.png">